packages/edit-site/src/components/global-styles/font-library-modal/utils/preview-styles.js

```diff
@@ -60,7 +60,8 @@ export function getFamilyPreviewStyle( family ) {
 	if ( family.fontFace ) {
 		//get all the font faces with normal style
 		const normalFaces = family.fontFace.filter(
-			( face ) => face.fontStyle.toLowerCase() === 'normal'
+			( face ) =>
+				face?.fontStyle && face.fontStyle.toLowerCase() === 'normal'
```

Comment:
This change was to prevent fatals as we're passing the function style variations without presets.